elyra/util/http.py

```
@@ -59,10 +59,12 @@ def write_error(self, status_code, **kwargs):
             else:
                 if isinstance(exception, Exception) and exception.args:
                     if isinstance(exception.args[0], Exception):
-                        reply["message"] = f"Error. The server sent an invalid response.\
+                        reply["message"] = (
+                            f"Error. The server sent an invalid response.\
                                 \nPlease open an issue and provide this error message,\
                                 any error details, and any related JupyterLab log messages.\
                                 \n\nError found:\n{str(exception.args[0])}"
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)